content/post/Dabbling_with_react_@_React_India_2019.md

```diff
@@ -0,0 +1,72 @@
+
+React India was the first ever international, community-led beach conference that provided a platform for developers to share, discuss their insights and experiences with React and React Native. The three-day conference was the first of its kind in India broken into workshops and conference days with community talks from 26th September - 28th September 2019
+
+Before setting off, I planned to capture a picture of everyone I met at the conference & tweet about it, that way when the conference ends. I could write this blog much more easily. And now you know why exactly it's called a tweet story. 
+
+## The Journey 
+
+The journey starts with me filling the diversity scholarship form for React India and in turn getting selected from a pool of hundreds. Happy to say, the other nine people who were selected were gems in their own fields. I had a marvelous time interacting with each and every one of them.
+
+I hopped on an early morning flight to Goa and was met with a brief patch of Goa rains. We boarded the airport shuttle organised by React India, filled with fellow attendees. From there on we started for the beautiful south side of Goa, along the narrow lanes of the countryside where we saw colorful houses, cottages behind tall coconut trees everwhere the eye could wander.
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">First up, catching up with <a href="https://twitter.com/bishnoi_pallavi?ref_src=twsrc%5Etfw">@bishnoi_pallavi</a> from <a href="https://twitter.com/CodingBlocksIn?ref_src=twsrc%5Etfw">@CodingBlocksIn</a> and <a href="https://twitter.com/manvisinghwal?ref_src=twsrc%5Etfw">@manvisinghwal</a> from <a href="https://twitter.com/Girlscript1?ref_src=twsrc%5Etfw">@Girlscript1</a> on our early morning flight to Goa. <br><br>Skies look clear, and time for takeoff. React India here we come 🐣🚨 <a href="https://t.co/Yt3ZvsGAGC">pic.twitter.com/Yt3ZvsGAGC</a></p>&mdash; Vipul Gupta 🐣 (@vipulgupta2048) <a href="https://twitter.com/vipulgupta2048/status/1177007858958753793?ref_src=twsrc%5Etfw">September 25, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
+
+## Workshops Galore
+
+The first day of the React India was all about workshops and connecting with individuals. I met professionals from Accenture, Verizon, Paperphile and made new connections while touring the amazing property of Planet Hollywood. It was a beach conference, why not enjoy it?. 
+
+There were many workshops happening that day, most notably of Siddharth's about Building your first Design System: from scratch to production and Vladmir Novik's workshop about Practical ReasonML for React devs.
```

Comment:
```suggestion
Many workshops were happening that day, most notably of Siddharth's about Building your first Design System: from scratch to production and Vladimir Novik's workshop about Practical ReasonML for React devs.
```

You sure his name is not spelled Vladimir?

Reply:
Extremely accurate. Thanks!